extensions/modules/mail/src/main/java/org/exist/xquery/modules/mail/SendEmailFunction.java

```diff
@@ -332,10 +340,14 @@ private boolean[] sendBySMTP(final Mail[] mails, final String smtpServerArg) thr
 
             //write SMTP message(s)
             for (int i = 0; i < mails.length; i++) {
```

Comment:
could use java5 styled loop :-P

Reply:
Yes, but that is existing code rather than new code that I added